packages/config/src/validation.ts

```diff
@@ -41,6 +41,33 @@ const isFalse = (value: any): boolean => {
   return value === false
 }
 
+type ValidationResult = ErrResult | boolean | string;
+type ValidationFn = (key: string, value: any) => ValidationResult
+
+export const validateAny = (...validations: ValidationFn[]): ValidationFn => {
+  return (key: string, value: any): ValidationResult => {
+    return validations.reduce((result: ValidationResult, validation: ValidationFn) => {
+      if (result === true) {
+        return result
+      }
+
+      return validation(key, value)
+    }, false)
+  }
+}
+
+export const validateAll = (...validations: ValidationFn[]): ValidationFn => {
+  return (key: string, value: any): ValidationResult => {
+    return validations.reduce((result: ValidationResult, validation: ValidationFn) => {
+      if (result !== true) {
+        return result
+      }
+
+      return validation(key, value)
+    }, true)
+  }
+}
+
```

Comment:
```suggestion
```

I don't see this used anywhere besides tests, so I'd rather not add it.

Reply:
Understood. Additionally, by forcing the flag to be a string or string[], you eliminate the need for this helper AND `validateAny`, so both can be removed.